charts/argo-rollouts/README.md

```diff
@@ -39,6 +39,7 @@ $ helm install --name my-release argo/argo-rollouts
 | controller.tolerations | [Tolerations for use with node taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) | `[]` |
 | controller.affinity | [Assign custom affinity rules to the deployment](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/) | `{}` |
 | controller.nodeSelector | [Node selector](https://kubernetes.io/docs/user-guide/node-selection/) | `{}` |
+| controller.enableMonitor | bool | `true` |  |
```

Comment:
I'm not a maintainer, but I believe it makes more sense to use the convention followed by other charts, so this would be `controller.metrics.serviceMonitor.enabled`. Maybe it also makes sense to add the full functionality like other charts with a [service monitor](https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/templates/argocd-server/servicemonitor.yaml) and full [metrics values block](https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml#L122-L134).

Reply:
Changed value name to follow argo-cd.